### PR TITLE
Convert column list from set to list.

### PR DESCRIPTION
**Pull request by agijsberts**
From version 1.5.0, Pandas no longer allows `set` object for the `index` or `columns` arguments of the `DataFrame` constructor. This PR converts the `set` to a `list` in `colname_unique` to accommodate this change.

Unfortunately, I do not have an IG account, so I could not run all unit tests.

### CODE REVIEW

File: trading_ig/rest.py [hunk 1]
```diff
@@ -866,8 +865,8 @@ def fetch_open_positions(self, session=None, version='2'):
 
         if self.return_dataframe:
 
-            list = data["positions"]
-            data = pd.DataFrame(list)
```
Comment (bug-or-feature) [inline trading_ig/rest.py:870]: What is the purpose of this change?

Comment (agijsberts) [reply trading_ig/rest.py:870]: Nothing fundamental, just that it is typically considered bad practice to name variables after builtins. This actually bit me, because my first dirty workaround was to simply cast the `columns` argument to `DataFrame` to `list`. Obviously, `list` no longer did what I expected.

For what it's worth, `lst` is also the variable name used at https://github.com/ig-python/ig-markets-api-python-library/blob/master/trading_ig/rest.py#L1038 in the same context.
